modules/install-secured-cluster-operator.adoc

```diff
@@ -29,7 +29,7 @@ You must install {product-title} `SecuredCluster` resource in its own project an
 . Enter the new project name as *stackrox* or some other name, and click *Create*.
 . Under the *Provided APIs* section, select *Create instance* on the *Secured Cluster* API.
 . Enter a name for your `SecuredCluster` custom resource.
-. For *Central Endpoint*, enter the address and port number of your Central instance. For example, if Central is available at `\https://central.example.com`, then specify the central endpoint as `central.example.com:443`. The default value of `central.stackrox:443` only works if you are installing secured cluster services in the same cluster that Central is installed in.
+. For *Central Endpoint*, enter the address and port number of your Central instance. For example, if Central is available at `\https://central.example.com`, then specify the central endpoint as `central.example.com:443`. The default value of `central.stackrox.svc:443` only works if you are installing secured cluster services in the same cluster that Central is installed in.
```

Comment:
For the "someday maybe" list: instead of "if you are installing" maybe "when you install".  However:
A) I'm not sure if it's only smoother to my ear.
B) It would be a pervasive change.

Maybe also "...secured cluster services and Central in the same cluster."